### PR TITLE
[7.4-only] LPS-123738 Move additionalProps inside its own property instead destructurint the content

### PR DESCRIPTION
**Pull request by carloslancha**
This pr is solving the issue that anything added as an `additionalProp` would be passed destructured to the component, and that might end being rendered as attributes.

See full conversation in https://liferay.slack.com/archives/G3JBR21HA/p1605725869270000

### CODE REVIEW

**Review by julien — APPROVED** [review]

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by julien** [comment]
Hey @carloslancha I see this



Does that mean this needs `gradlew baseLine` or similar?

File: modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/js/ActionsComponentPropsTransformer.js [hunk 1]
```diff
@@ -52,10 +52,9 @@ const ACTIONS = {
 };
 
 export default function propsTransformer({
-	deleteVocabulariesURL,
+	additionalProps: {deleteVocabulariesURL, viewVocabulariesURL},
```
Comment (jbalsas) [inline modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/js/ActionsComponentPropsTransformer.js:55]: This is handy!

**Comment by jbalsas** [comment]
Failures seem related to a possible DDM migration... so going to forward this...

/cc @matuzalemsteles, @diegonvs, just in case you want to check [ContentPagesWithMapping#ViewMappableFieldsAreGroupedByFieldset](https://testray.liferay.com/reports/production/logs/test-1-15/1605810663884/test-portal-acceptance-pullrequest(master)/3538/functional-tomcat90-mysql57-jdk8/0/13/LocalFile.ContentPagesWithMapping_ViewMappableFieldsAreGroupedByFieldset/index.html.gz) or [ContentPagesWithEditMode#OnlyCanEditContentWhenSelectContentEdition](https://testray.liferay.com/reports/production/logs/test-1-15/1605810663884/test-portal-acceptance-pullrequest(master)/3538/functional-tomcat90-mysql57-jdk8/0/10/LocalFile.ContentPagesWithEditMode_OnlyCanEditContentWhenSelectContentEdition/index.html.gz)

**Comment by jbalsas** [comment]
ci:forward

**Comment by diegonvs** [comment]
> Failures seem related to a possible DDM migration... so going to forward this...

This failures are unrelated to our changes. I'll ping 🇧🇷 QAs

**Pull request closed without merging**